clients/gateway.go

```diff
@@ -210,9 +233,39 @@ type Abi []struct {
 		Name string `json:"name"`
 		Type string `json:"type"`
 	} `json:"inputs"`
-	Outputs []interface{} `json:"outputs"`
+	Outputs []struct {
+		Name string `json:"name"`
+		Type string `json:"type"`
+	} `json:"outputs"`
+	StateMutability string `json:"stateMutability"`
+	Data            []struct {
+		Name string `json:"name"`
+		Type string `json:"type"`
+	}
+	Keys []interface{} `json:"keys"`
 }
 
+// // get_code
+// type CodeDefinition struct {
+// 	Bytecode []*felt.Felt `json:"bytecode"`
+// 	Abi      Abi          `json:"abi"`
+// }
+
+// type Program struct {
+// 	Builtins         []string      `json:"builtins"`
+// 	Prime            string        `json:"prime"`
+// 	ReferenceManager interface{}   `json:"reference_manager"`
+// 	Identifiers      interface{}   `json:"identifiers"`
+// 	Attributes       []interface{} `json:"attributes"`
+// 	Data             []*felt.Felt  `json:"data"`
+// 	DebugInfo        interface{}   `json:"debug_info"`
+// 	MainScope        interface{}   `json:"main_scope"`
+// 	Hints            interface{}   `json:"hints"`
+// 	CompilerVersion  string        `json:"compiler_version"`
+// }
```

Comment:
Not sure why these are commented out.

Reply:
my bad. i forgot to delete it. thanks for catching this